### ISSUE TITLE
PS7: Pipeline Issues

### ISSUE THREAD

**Issue by JustinGrote** [lordmilko/PrtgAPI#136]
**Describe the bug**
*Clearly and concisely describe what you were trying to do, what happened and what you were expecting to happen*

Trying to pipe commands consistently produces `Set-ChannelProperty: Don't know how to extract the pipeline input from a 'System.SZArrayEnumerator' enumerator from type ''.`

Specifying the items directly usually works fine.

![image](https://user-images.githubusercontent.com/15258962/76458846-693cea00-6398-11ea-80c8-d21568640f40.png)

**Steps to reproduce**
*Put the relevant code from your application that caused the issue to happen in the code block below*

```powershell
$pingtimechannels | set-channelproperty -LimitsEnabled $true -WarningLimitMessage "Excessive Latency detected for this device. Verify sufficient bandwidth and connectivity to the site." -UpperWarningLimit 300
```

**What is the output of `Get-PrtgClient -Diagnostic`?**

```powershell
# Output goes here!
PSVersion      : 7.0.0
PSEdition      : Core
OS             : Microsoft Windows 10.0.19041
PrtgAPIVersion : 0.9.11
Culture        : en-US
CLRVersion     : .NETCoreApp,Version=v3.1
PrtgVersion    : 20.1.56.1547
PrtgLanguage   : english.lng

```

**Additional context**
*Anything else I should know to help solve the issue?*
Probably just a type difference or .NET Core 3.1 regression.

**Comment by lordmilko** [comment]
Hi @JustinGrote,

I was unable to reproduce this issue; are you able to provide the full code for how you initialized the `$pingtimechannels` variable?

**Comment by lordmilko** [comment]
Hi @JustinGrote,

I've managed to replicate this and have released a pre-release build of PrtgAPI 0.9.13 that includes a fix for this issue. Can you please follow the [manual installation instructions](https://github.com/lordmilko/PrtgAPI/wiki/Installation#manual) and advise whether this build resolves the issue?

**Comment by JustinGrote** [comment]
Worked for me, thanks!
